frontend/src/pages/forgotPassword/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Form, Header } from '../../components';
+import { Container, Wrapper } from '../../components/layout/layout.styles';
+import { Title } from '../../components/text/text-styles';
+import EmailFormValidation from './formValidation';
+
+const ForgotPassword = () => (
+  <>
+    <Header hideProfile />
+    <Wrapper
+      flexDirection={['column', 'row']}
+      alignItems="center"
+      justifyContent="center"
+    >
+      <Container width={[1, 1 / 2]}>
+        {' '}
```

Comment:
{' '}?

Reply:
That's something prettier adds for some reason.